rest/oidc_api.go

```diff
@@ -81,7 +108,14 @@ func (h *handler) handleOIDCCommon() (redirectURLString string, err error) {
 	}
 
 	var redirectURL *url.URL
-	state := ""
+
+	// Set state parameter to prevent cross-site request forgery (CSRF).
+	state := strings.Replace(uuid.New().String(), "-", "", -1)
+	http.SetCookie(h.response, &http.Cookie{
```

Comment:
Do OIDC state cookies need to have any of SameSite, Secure or HTTPOnly set?

I think Secure can be enabled similar to session cookies via the SecureCookieOverride database context option.

I spent 10 mins searching but didn't find anything definitive for SameSite or HTTPOnly.
Do you know of any other implementations of state cookie parameters you can cross-check?